packages/keycodes/src/test/index.js

```diff
@@ -230,3 +231,273 @@ describe( 'rawShortcut', () => {
 		} );
 	} );
 } );
+
+describe( 'isKeyboardEvent', () => {
+	afterEach( () => {
+		while ( document.body.firstChild ) {
+			document.body.removeChild( document.body.firstChild );
```

Comment:
I was wondering if removing DOM element will also remove all attached listeners. It looks like the answer is yes:

https://stackoverflow.com/questions/12528049/if-a-dom-element-is-removed-are-its-listeners-also-removed-from-memory

I thought I will share it for others looking at this PR who weren't sure.